laravel/resources/views/layout.blade.php

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="de">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>@yield('site_title')</title>
```

Comment:
[SUGGESTION]

Instead of defining the application name hardcoded in every blade:

```suggestion
    <title>@yield('site_title') | {{ config('app.name', 'TeamSpeak Dynamic Banner') }}</title>
```